### DISCUSSION TITLE
Lock customer account

### DISCUSSION THREAD

**Discussion by cryexx** [Blair2004/NexoPOS#1325] [Ideas]
If there is a problem with the customer account, it would be nice to be able to block the customer account from purchases, for example, via a checkbox in the Customers section.

**Environment:**
 - NexoPOS 4.8.14

**Comment by Blair2004** [comment]
Customer being a user resource, this can now be banned. On NexoPOS 5x, banned customers can't be selected for sale.
![image](https://user-images.githubusercontent.com/5265663/221685511-09c3a32a-1e5e-40f5-84e4-133f3d77b87b.png)
